GCRCatalogs/dc2_forced_source.py

```diff
@@ -3,61 +3,13 @@
 """
 
 import os
-import re
-import warnings
-import shutil
-
-import numpy as np
-import pyarrow.parquet as pq
-import yaml
-
-from GCR import BaseGenericCatalog
 
+from .dc2_dm_catalog import DC2DMCatalog, convert_flux_to_nanoJansky
```

Comment:
```suggestion
from .dc2_dm_catalog import DC2DMCatalog, convert_flux_to_nanoJansky, create_basic_flag_mask
```
`create_basic_flag_mask` is missing